packages/common/src/utils/NumUtils.ts

```diff
@@ -0,0 +1,5 @@
+export const NumUtils = {
+  isPositive: (num: number): boolean => {
+    return num > 0;
+  },
+};
```

Comment:
Is this silly?

Idea being colocation of general number utils 🤷

Reply:
it's a good pattern - not silly I think.
I do wonder about moving the 'is low months of stock' into here actually - could be useful to have in one place